logstash-core-event-java/lib/logstash-core-event-java_jars.rb

```diff
@@ -1,6 +1,14 @@
 # this is a generated file, to avoid over-writing it just delete this comment
```

Comment:
why change this file? this is a generated file. If jar_dependencies is not loaded the problem is probably elsewhere